starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+  <link rel="stylesheet" type="text/css" href="css/header.css">
+
```

Comment:
in your head you need to add this : 

`    <meta charset="UTF-8">`

Without this, your `-` appear as `â€“`